fvtest/threadtest/keyDestructorTest.cpp

```diff
@@ -0,0 +1,117 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 1991, 2016
```

Comment:
Copied code so it has to take the earliest copyright from the file I copied from.